mrtt-ui/README.md

```diff
@@ -1,50 +1,49 @@
-# Getting Started with Create React App
+# Mangrove Restoration Tracking Tool (MRTT) User Interface
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+## Product Description and Motivation
 
-## Available Scripts
+Product description and motivation coming soon
 
-In the project directory, you can run:
+You can view the data that the MRTT app creates [here](https://globalmangrovewatch.org/).
 
-### `npm start`
+The project board for this work can be found [here](https://github.com/Vizzuality/mangrove-atlas/issues)
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+## Tech Stack and Dependencies Worth Noting
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+- React
+- Emotion to manage styling
+- MUI for user already styled user interface components
+- React-toastify
+- React Hook Form
+- Yup for form validation
+- Eslint, Prettier, and Husky to manage git hooks
+- Mapbox to draw mangrove site locations.
+- [The Global Mangrove Watch API](https://github.com/globalmangrovewatch/gmw-api) to handle data on the server side
+- Transifex for user language translation.
 
-### `npm test`
+## Archetectural Decisions and Tradeoffs
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+- Translation will be provided via Transifex which is configured at `mrtt-ui/public/index.html`. This is unlikely to translate dynamically generated text, so we included a language file (`mrtt-ui/src/language.js`)for easy future translation. We attempted to put all user facing text in the language file, but in some places it is hardcoded in the JSX
```

Comment:
nice job adding this note!

Reply:
I only remembered cause you mentioned it, lol